android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/ExposureNotificationClientWrapper.java

```diff
@@ -120,18 +119,15 @@ public void showPermissionDialog(Activity activity, ApiException apiException) {
         return exposureNotificationClient.getExposureSummary(token);
     }
 
-    /**
-     * Gets the {@link List} of {@link ExposureInformation} using the stable token.
-     */
-    public Task<List<ExposureInformation>> getExposureInformation(String token) {
-        return exposureNotificationClient.getExposureInformation(token);
+    public Task<List<ExposureWindow>> getExposureWindows() {
+        return exposureNotificationClient.getExposureWindows(ExposureNotificationClient.TOKEN_A);
```

Comment:
TOKEN_A enables ExposureWindow mode in API v1.5
https://developers.google.com/android/exposure-notifications/exposure-notifications-api